contrib/dockerfile_build.bzl

```diff
@@ -106,6 +110,10 @@ dockerfile_image = repository_rule(
             doc = "A map of args to pass to the --build-arg option in the " +
                   "`docker build` command.",
         ),
+        "cache": attr.bool(
+            default = False,
+            doc = "Turn on docker caching for the build.",
```

Comment:
hmm... Lets add a big warning that this will mess up with Bazel caching mechanisms and in case of failures / issues you might need to do a bazel clean --expunge_async + do a docker cache purge.